### PR TITLE
Add Gleam

### PR DESCRIPTION
**Pull request by Yang-Nankai**
Gleam.io is a platform that helps businesses and marketers create and manage online competitions, giveaways, and promotions. It provides tools for driving engagement, collecting leads, and growing audiences through customizable entry forms, reward programs, and social media integrations. Gleam.io offers 2FA for the account login verification process. The website is ranked 14,146 on [Similarweb](https://pro.similarweb.com/#/digitalsuite/websiteanalysis/overview/website-performance/*/999/1m?webSource=Total&key=gleam.io). The website provides documentation on how to turn on 2FA.

### CODE REVIEW

File: entries/g/gleam.io.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "Gleam": {
+    "domain": "gleam.io",
+    "tfa": [
+      "totp",
+      "custom-software"
+    ],
+    "custom-software": [
+      "Authy"
```
Comment (fletcherdodd) [inline entries/g/gleam.io.json]: Do you know if Gleam uses "[Authy Powered Tokens](https://help.twilio.com/articles/19753389187355)" or just authenicator tokens? If it's the latter then we can drop the custom software from the listing as they were probably just suggesting it as a 2FA app in the documentation.

Comment (hkamran80) [inline entries/g/gleam.io.json:9]:
```suggestion
      "totp"
```

They only offer TOTP.

**Review by hkamran80 — CHANGES_REQUESTED** [review]

File: entries/g/gleam.io.json (hunk 1 above)
Comment (Yang-Nankai) [reply entries/g/gleam.io.json]: After testing it, I found it to be the latter.

image

Comment (fletcherdodd) [reply entries/g/gleam.io.json]: Nice work! I assume you have reset your 2FA after sending that screenshot (if not I would). But just a tip for the future, I would blur out any sensitive data like 2FA setup QR codes, email addresses, etc. when providing evidence in a PR or issue on here.

**Review by fletcherdodd — APPROVED** [review]

**Pull request merged**